src/biokbase/narrative/app_util.py

```diff
@@ -162,6 +99,31 @@ def _untransform(transform_type, value):
 
 
 def app_param(p):
+    """
```

Comment:
Added a bit of notes here that weren't exactly intended for this PR, though still helpful.